README.md

```diff
@@ -221,6 +221,12 @@ To run a single test, add `.only` into the name of any `describe` or `it` functi
 `describe.only('Skip Conditions', function() {...}` or
 `it.only('Given this is a test', function() {...}`
 
+In order to speed up our functional tests running in Github Actions on pull request, the tests have been split out into 4 suites which can be run concurrently. These suites are configured in the `wdio.conf.js` file.
+An individual test suite can be run using:
```

Comment:
I don't think the GitHub actions bit is relevant to document here. It thinks it just needs to state that we have suites and you can run them using etc.